components/modals/ErrorModal.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { Dialog } from "@headlessui/react"
+
+export const ErrorModal = () => <div>
+  <Dialog.Title as="h3"
+    className="m-5 text-lg leading-6 font-medium text-gray-900">
+    Oops
```

Comment:
Consider rolling back to block-pagination compatible error modal, displaying which block contains no bundle

Reply:
What is considered a block-pagination compatible error modal?